src/test/suite/fixtures/recorded/surroundingPair/parseTree/clojure/clearBound.yml

```diff
@@ -0,0 +1,27 @@
+languageId: clojure
```

Comment:
Checking that we can remove parens, tho in reality you'd prob say "chuck bound" instead, to remove without moving cursor.  You'll prob also want to try eg "box repack pair" to change parens to square brackets